src/ui/public/management/index.js

```diff
@@ -0,0 +1,23 @@
+import ManagementSection from './section';
+
+const sections = new ManagementSection('management', {
+  display: 'Management'
+});
+
+// TODO: where should this live?
+sections.register('data', {
+  display: 'Connect Data',
+  order: 0
+});
+
+sections.register('elasticsearch', {
+  display: 'Elasticsearch',
+  order: 10
+});
+
+sections.register('kibana', {
+  display: 'Kibana',
+  order: 20,
+});
```

Comment:
I would like to discuss the interface for registering a section. Currently, it's required that you register a section prior to adding items to it. That has proved to be challenging as everything is implemented as a plugin with no guaranteed order. One thought is by getting a section will create it if it doesn't exist. You could then set details like order and display name elsewhere. Because of this challenge, I am currently initializing the core sections here, any empty section will be ignored in the UI.